packages/react/src/__helpers__/jest.setup.ts

```diff
@@ -0,0 +1,18 @@
+import { Blob } from 'buffer';
+import crypto from 'crypto';
+import { TextEncoder, TextDecoder } from 'util';
+
+Object.defineProperty(global, 'crypto', {
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+  value: Object.setPrototypeOf({ subtle: crypto.subtle }, crypto),
+});
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+global.Blob = Blob;
+
+// until https://github.com/jsdom/jsdom/issues/2524 is resolved
+global.TextEncoder = TextEncoder;
```

Comment:
where are these used?

Reply:
Inside https://github.com/lens-protocol/lens-sdk/pull/192/files#diff-fa678de0ca5e45b91b0e063a33b6fd1db971e0a4ac7d43059ac3adfb9b41513e from a previous PR.

As in this PR i am testing a component thar uses `GatedClient`, it now needs the same polyfills.